.github/workflows/validate-schemas.yml

```
@@ -0,0 +1,26 @@
+        steps:
+            - uses: actions/checkout@v4
+
+            - name: json-yaml-validate
+              id: json-yaml-validate
+              uses: GrantBirki/json-yaml-validate@v3.3.2
+              with:
+                  mode: "warn"
+                  comment: true
+                  json_schema_version: "draft-2020-12"
+                  json_extension: ".schema.json"
+                  base_dir: ./schema
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)